complaints/views.py

```diff
@@ -104,8 +132,80 @@ def handle_unblock_request(request):
                 messages.success(
                     request, "The Complaint has been Successfully Resolved"
                 )
+                status = "Resolved"
             if request.POST.get("status") == Complaint.CANCELLED:
                 messages.success(
                     request, "The Complaint has been Successfully Cancelled"
                 )
+                status = "Cancelled"
+            email_resolve(
+                category=str(form_obj.category),
+                request_id=str(form_obj.id),
+                issue="Request",
+                user_email=str(request.user.email),
+                request_status=status,
+                details=str(form_obj.reason),
+                remark_user=str(form_obj.remark_to_user),
+            )
             return render(request, "registration/home.html", context={"title": "home"})
+
+
+def email_on_request(request_id, category, details, issue, user_email):
+    """Function to send email to the user and staff when he registers a new complaint or request"""
+
+    from_email = settings.EMAIL_HOST_USER
+    subject = issue + " Registered"
+    message = (
+        "Your "
+        + issue
```

Comment:
The `+` operator isn't required when enclosed by parentheses. Look into using format strings instead of manually concatenating like this.